conf/broker.conf

```diff
@@ -1309,13 +1309,11 @@ replicatorPrefix=pulsar.repl
 replicationPolicyCheckDurationSeconds=600
 
 # Default message retention time.
-# The default value is 0, which means the data is removed after all the subscriptions are consumed.
-# Value less than 0 means messages never expire.
+# Value of 0 means the retention is disabled, -1 means the data will not be removed by time quota.
```

Comment:
```suggestion
# 0 means retention is disabled. -1 means data is not removed by time quota.
```

Write in the simple present tense as much as possible if you are covering facts that were, are, and forever shall be true.
https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.e8uqh1awkcnp

Reply:
Updated. PTAL @Anonymitaet